faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/feature/quran/sur/SurEffect.kt

```diff
@@ -0,0 +1,7 @@
+package net.thechance.mena.faith.presentation.feature.quran.sur
+
+sealed interface SurEffect {
+    data object BackNavigation : SurEffect
+    data object BookmarkNavigation : SurEffect
+    data class SurahDetailsNavigation(val surahId: Int) : SurEffect
```

Comment:
what do you think?  it's clearer that this is a navigation effect 
```suggestion
    data object BackNavigation : SurEffect
    data object NavigateToBookmark : SurEffect
    data class NavigateToSurahDetails(val surahId: Int) : SurEffect
```

Reply:
yes, but  is it better to be verb or a noun ?